code-challenges.test.js

```diff
@@ -1,5 +1,10 @@
 // ASSESSMENT 2: Coding Practical Questions with Jest
 
+const { array } = require("yargs")
```

Comment:
Remember to comment out these yargs helper code from vs code. This happens automatically when building tests. When you get errors just scroll to the top to find code you didn't create and comment out it out.